samples/splitter/splitter.cpp

```diff
@@ -560,17 +558,19 @@ void MySplitterWindow::OnPositionResize(wxSplitterEvent &event)
     // if it is not locked. Otherwise we hold it at the position
     // the user specified by manually dragging.
     if (m_frame->IsSashLocked())
+    {
+        // We set the last known position to keep the sash in place.
+        // For this particular example we could also simply use
+        // event.Veto()
+        // as well and it would have the same effect.
         event.SetSashPosition(m_frame->GetSashPos());
-
-    event.Skip();
+    }
 }
 
 void MySplitterWindow::OnPositionChanging(wxSplitterEvent& event)
 {
     wxLogStatus(m_frame, "Position is changing, now = %d (or %d)",
                 event.GetSashPosition(), GetSashPosition());
-
-    event.Skip();
```

Comment:
Sorry, why was this removed? It should stay, we're not really processing the event, just logging it, so skipping it makes perfect sense.

Reply:
I'm going to readd this and merge if you don't have any objections.

Reply:
No objection. Since the value is not changed, it doesn't matter if it is skipped or not. Maybe you should add a comment to that as well.